### DISCUSSION TITLE
Support for Indian Stocks

### DISCUSSION THREAD

**Discussion by devfinwiz** [kernc/backtesting.py#659] [General]
Backtest.py is capable of providing backtesting results for NSE listed stocks?

**Comment by pranavlal** [comment]
Hi,
Not directly therefore my approach is to build the data frame with the relevant data outside the backtesting framework and then feed it to backtesting. This is the main advantage of this library. It is very easy to fit in your own data.

I have begun using the library just yesterday so I may be wrong about direct  Indian stock support.

**Reply by devfinwiz** [reply]
Kind of right, yes.

**Reply by preritdas** [reply]
You supply your own data to the backtester. As long as your data has the Open, High, Low, and Close columns, you're good to go. Volume and other columns are optional. It doesn't matter where you get your data from.
